.github/workflows/ci.yml

```diff
@@ -105,14 +105,18 @@ jobs:
           toolchain: stable
           override: true
 
+      - name: Install cargo-nextest
+        uses: baptiste0928/cargo-install@v1
+        with:
+          crate: cargo-nextest
```

Comment:
we could also use the pre-built binaries https://nexte.st/book/release-urls.html

Reply:
Yep, it makes sense! Have changed to reflect: https://nexte.st/book/pre-built-binaries.html#using-pre-built-binaries-in-ci